src/SecureStore.Utils/SecureStore.Utils.projitems

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Remove this since we can not share Localization Resources in Shared Projects, my bad.